### PR TITLE
final changes

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is to address https://github.com/LibertyDSNP/frequency/issues/1153

### CODE REVIEW

File: pallets/stateful-storage/src/stateful_child_tree.rs [hunk 1]
```diff
@@ -377,9 +377,9 @@ impl<H: MultipartKeyStorageHasher> StatefulChildTree<H> {
 		storage_name: &[u8],
 	) -> ChildInfo {
 		let trie_root = Self::get_tree_prefix(msa_id);
-		// child tree root should be hashed by Twox128 to avoid probability of conflict
+		// child tree root should be hashed by Blake128 to avoid probability of conflict
```
Comment (aramikm) [inline pallets/stateful-storage/src/stateful_child_tree.rs:380]: changing to Blake to use cryptographic hash

**Review by saraswatpuneet — APPROVED** [review]
lgtm

File: pallets/stateful-storage/src/stateful_child_tree.rs (hunk 1 above)
Comment (JoeCap08055) [reply pallets/stateful-storage/src/stateful_child_tree.rs:380]: Do we really need it? Is it worth the expense?

Comment (aramikm) [reply pallets/stateful-storage/src/stateful_child_tree.rs:380]: From my perspective not but Parity recommended to use a Cryptographic hash function for child tree root. I don't think the expense is that bad compared to overall weights.

**Pull request merged**